kodilla-testing/src/main/java/com/kodilla/testing/forum/ForumUser.java

```diff
@@ -0,0 +1,78 @@
+package com.kodilla.testing.forum;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+public class ForumUser {
+    private String name;
+    private String realName;
+    private ArrayList<ForumPost> posts = new ArrayList<ForumPost>();
+    private LinkedList<ForumComment> comments = new LinkedList<ForumComment>();
+
+    public ForumUser(String name, String realName) {
+        //name visible on forum
+        this.name = name;
+        //real name of the user
+        this.realName = realName;
+    }
+
+    public void addPost(String author, String postBody){
+        ForumPost thePost = new ForumPost(postBody, author);
+        posts.add(thePost);
+    }
+
+    public void addComment(ForumPost thePost, String author, String commentBody){
+        ForumComment theComment = new ForumComment(thePost, commentBody, author);
+        comments.add(theComment);
+    }
+
+    public int getPostsQuantity(){
+        return posts.size();
+    }
+
+    public int getCommentsQuantity(){
+        return comments.size();
+    }
+
+    public ForumPost getPost(int postNumber){
+        ForumPost thePost = null;
+        if (postNumber >= 0 && postNumber < posts.size()) {
+            thePost = posts.get(postNumber);
+        }
+        return thePost;
+    }
+
+    public ForumComment getComment(int commentNumber){
+        ForumComment theComment = null;
+        if (commentNumber >= 0 && commentNumber < comments.size()){
+            theComment = comments.get(commentNumber);
+        }
+        return theComment;
+    }
+
+    public boolean removePost(ForumPost thePost){
+        boolean result = false;
+        if (posts.contains(thePost)){
+            posts.remove(thePost);
+            result = true;
+        }
+        return result;
+    }
+
+    public boolean removeComment(ForumComment theComment){
```

Comment:
tak jak [tu](https://github.com/robert-er/kodilla-course-m6/pull/2/files#r439820664)

Reply:
Klasy ForumComment, ForumUser i ForumPost są skopiowane z treści kursu. Wychodziłem z założenia, że jak jakiś przykład jest z kursu to nie trzeba go poprawiać :)